release-tools/src/main/resources/application-jenkins.properties

```diff
@@ -0,0 +1,30 @@
+git.username=${GIT_USERNAME}
+git.author=${GIT_AUTHOR}
+git.email=${GIT_EMAIL}
+git.password=${GIT_PASSWORD}
+
+github.api.url=${GITHUB_API_URL}
+
+deployment.username=${DEPLOYMENT_USERNAME}
+deployment.password=${DEPLOYMENT_PASSWORD}
+deployment.api-key=${DEPLOYMENT_API_KEY}
+deployment.maven-central.stagingProfileId=${STAGING_PROFILE_ID}
+deployment.settings-xml=${SETTINGS_XML}
+
+jira.username=${JIRA_USERNAME}
+jira.password=${JIRA_PASSWORD}
+jira.url=${JIRA_URL}
+
+gpg.keyname=9A2C7A98E457C53D
```

Comment:
Having all the other properties externalized, let's do the same for the keyname.